### ISSUE TITLE
Sync common files

### ISSUE THREAD

**Issue by donny-dont** [dart-lang/mono_repo#21]
When working with multiple packages its a pain to keep files in sync. For example you often want the same analysis_options.yaml across packages. It would be great to have a consistent way of doing this.

**Comment by donny-dont** [comment]
Other examples of files to keep in sync, .gitignore, LICENSE.

**Comment by zoechi** [comment]
you can have one .gitignore in the root directory
but otherwise I agree

**Comment by donny-dont** [comment]
It was my understanding that pub uses the .gitignore to determine what not to push

**Comment by zoechi** [comment]
Right, I remember that discussions

**Comment by matanlurey** [comment]
I'd like to see something like this!

**Comment by donny-dont** [comment]
@matanlurey so hypothetically how should this be approached?

Maybe a `.mono_repo` directory that has the files to sync?

**Comment by matanlurey** [comment]
Yeah I guess it depends if you want to configuration or convention based.

@kevmoo what say you?

**Comment by kevmoo** [comment]
Or just have an entry in `mono_repo.yaml` (after https://github.com/dart-lang/mono_repo/pull/29) that specifies a source project directory + files that should sync everywhere?
